recipes/quast/build.sh

```diff
@@ -16,7 +16,7 @@ QUAST_HOME=$PREFIX/opt/quast-$PKG_VERSION
 mkdir -p $BINARY_HOME
 mkdir -p $QUAST_HOME
 
-python "setup.py" install_full
+python "setup.py" install
```

Comment:
Beware that `quast` behaves very "non-standard-ish": As is, you need its custom `install_full` argument here to instruct it to build its dependencies. The build failure you got earlier is likely because with the change to `install` it now tries to compile `gnuplot` etc. during its run time but then misses the required compilers.
If you'd want `quast` to behave more conventional (which is likely the preferable long-term solution), you would have to patch it to avoid the custom compilation steps, i.e., patch/look at `setup.py` and *all* files that use `LIBS_LOCATION` from `quast_libs/qconfig.py`..

Reply:
Hi @mbargull, I've changed `install_full` to `install` here just for the sake of speed. Note that both options process `gnuplot` compilation equal likely, the difference between them is in few other modules (stuff for structural variations and without reference mode of MetaQuast -- i.e. relatively rarely needed). I plan to return `install_full` back when we solve the main issue with the build failure. Note that the error is absolutely the same and I initially faced it after my first commits in this patch where I used `install_full` as before.

As for the general solution -- I will try to patch Quast accordingly.
However, I still do not understand what is going wrong with the build right now. Quast v.4.6.1 is pretty similar to v.4.5 which was successfully added to bioconda. All differences are mostly cosmetics and should not affect compilation/installation at all..

Reply:
Oh, wow, my bad about the `install`/`install_full` thing! I must've read something wrong while skimming `setup.py`, sorry.
But it looks like the `gnuplot` directory isn't added to `package_data`. Maybe adding a `find_package_files('gnuplot')` in there will fix the current `gnuplot` problem.

Reply:
Well, it was our fault, I've added a patch to include `find_package_files('gnuplot')` in `setup.py`. Now, Quast does not try to recompile gnuplot in the test run as it did previously, which is good.
However, it has not solved the main issue. Now I don't have any more clues why the test is failing :)  Or why `; cat ....stderr` in `meta.yaml` is not working.

Reply:
> Well, it was our fault

Ah, I didn't even notice before, I was talking to one of the tool's devs!

To look into what failed during the `gnuplot` compilation, you'd have to inspect `quast_libs/gnuplot/make.err`, I'd guess from https://github.com/ablab/quast/blob/quast_4.6.1/quast_libs/ca_utils/misc.py#L132.
Also, I would suggest to let the build fail if some compilations failed, e.g., at https://github.com/ablab/quast/blob/quast_4.6.1/setup.py#L271 insert via a patch something like
```python
if not_installed_message:
    raise RuntimeError(not_installed_message)
```
since we really want to avoid client-side compilation and get a readily build package from `conda-build`.
[Either way, if you want to integrate Quast properly into the Conda/Bioconda ecosystem (and avoid those `gnuplot` build issues :wink: overall), the best thing would be to avoid/patch out all custom compilation steps and let `conda` install all dependencies]

Reply:
Well, I committed one more patch, so we now reuse system `gnuplot` if it is available. So, no there is no warnings/notices/etc about any problems with it. However, the test is still failing!

Reply:
Maybe the test went fine and just didn't create the `.stderr` file you are trying to read and thus `cat` fails? Just a hunch.
But it's great that an external `gnuplot` can now be used, thanks! Do you think it would be possible to do similar patches for some of the other tools? Looking inside `quast_libs`, I see a couple of programs that are available on `bioconda`, e.g., `bwa`, `bedtools`, `glimmer`, `gnuplot`, `mummer`, `sambamba`.

Reply:
Well, initially I did not use `cat` but the test was crashing and I was curios what is happening. The glue is probably in this .stderr file, so Björn (@bgruening) suggested me to add `cat`. However, it is not printed anyway :( 
Note: this files is always created until something extraordinary has happened. But again, it is not the source of problem because it is the same as before adding `cat`.